pkgs/by-name/gu/guile-ssh/package.nix

```diff
@@ -12,22 +11,16 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "guile-ssh";
-  version = "0.16.3";
+  # XXX: using unstable to ensure proper build with libssh 0.11.1 (https://github.com/artyom-poptsov/guile-ssh/issues/42)
+  version = "unstable-2024-10-15";
 
   src = fetchFromGitHub {
     owner = "artyom-poptsov";
     repo = "guile-ssh";
-    rev = "v${finalAttrs.version}";
-    hash = "sha256-P29U88QrCjoyl/wdTPZbiMoykd/v6ul6CW/IJn9UAyw=";
+    rev = "9336580f92f83bb73041c5374b400144a56b4c35";
+    hash = "sha256-Hwg0xaNSm/SEZfzczjb7o8TJXfzT1mmOk1rJROxahLQ=";
   };
 
-  patches = [
-    (fetchpatch {
-      url = "https://github.com/artyom-poptsov/guile-ssh/pull/31/commits/38636c978f257d5228cd065837becabf5da16854.patch";
```

Comment:
This might still be needed from what I read on https://github.com/artyom-poptsov/guile-ssh/pull/31, any reason for removing it?

ignore this comment if irrelavent, I don't know much about guile-ssh or guix.

Reply:
My mistake, I removed the patch because I thought it corresponded to a commit contained by the latest unstable.